cpp/include/cudf/column/column_view.hpp

```diff
@@ -447,6 +455,19 @@ class column_view : public detail::column_view_base {
     return device_span<T const>(data<T>(), size());
   }
 
+ protected:
+  // TODO: Fix noexcept, see notes in prefetch.hpp
```

Comment:
This has been addressed and I've removed the comment locally, but I'm waiting to push until a build completes so that artifacts will be available for benchmarking.